pkgs/development/php-packages/castor/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, installShellFiles
+, php
+, nix-update-script
+, testers
+, castor
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "castor";
+  version = "0.8.0";
+
+
+  src = fetchurl {
+    url = "https://github.com/jolicode/castor/releases/download/v${finalAttrs.version}/castor.linux-amd64.phar";
+    sha256 = "sha256-0lnn4mS1/DgUoRoMFvCjwQ0j9CX9XWlskbtX9roFCfc=";
```

Comment:
```suggestion
    hash = "sha256-0lnn4mS1/DgUoRoMFvCjwQ0j9CX9XWlskbtX9roFCfc=";
```

Reply:
Rahh I always forget this because I generate the template with nix-init :sweat_smile: